pyracing/helpers.py

```diff
@@ -7,7 +7,10 @@
 
 # iRacing has all of their timestamps in ms so we need to divide
 def datetime_from_iracing_timestamp(timestamp):
-    return datetime.utcfromtimestamp(int(timestamp) / 1000)
+    if type(timestamp) is int or type(timestamp) is float:
+        return datetime.utcfromtimestamp(int(timestamp) / 1000)
+    else:
```

Comment:
No need for else here, just return outside the if

Reply:
```suggestion
    
```
This line gets deleted completely.